src/helpers/agGrid.ts

```diff
@@ -235,31 +235,63 @@ export const getFields = (dataHeaders: Execution.IResultHeaderItem[][]) => {
     }) as string[];
 };
 
+const getSubtotalLabelCellIndex = (resultHeaderItems: Execution.IResultHeaderItem[][], rowIndex: number): number => {
+    return resultHeaderItems.findIndex(headerItem => Execution.isTotalHeaderItem(headerItem[rowIndex]));
+};
+
+const getCell = (
+    rowHeaderData: Execution.IResultHeaderItem[][],
+    rowIndex: number,
+    rowHeader: IGridHeader,
+    rowHeaderIndex: number,
+    intl: InjectedIntl
+) => {
+    const rowHeaderDataItem = rowHeaderData[rowHeaderIndex][rowIndex];
+    const cell = {
+        field: rowHeader.field,
+        rowHeaderDataItem
+    };
+
+    if (Execution.isAttributeHeaderItem(rowHeaderDataItem)) {
+        return {
+            ...cell,
+            value: rowHeaderDataItem.attributeHeaderItem.name
+        };
+    }
+
+    if (Execution.isTotalHeaderItem(rowHeaderDataItem)) {
+        const totalName = rowHeaderDataItem.totalHeaderItem.name;
+        return {
+            ...cell,
+            value: getSubtotalLabelCellIndex(rowHeaderData, rowIndex) === rowHeaderIndex
+                ? intl.formatMessage({ id: `visualizations.totals.dropdown.title.${totalName}` })
+                : null
+        };
+    }
```

Comment:
missing invariant to exclude `isMeasureHeaderItem`

Reply:
Done